backends/libpurple/main.cpp

```diff
@@ -1673,9 +1668,10 @@ void * requestInput(const char *title, const char *primary,const char *secondary
 			np->m_inputRequests[req->mainJID] = req;
 			return NULL;
 		}
-		else if (primaryString == "Set your Steam Guard Code") {
-			LOG4CXX_INFO(logger, "prpl-steam-mobile steam guard request");
-			np->handleMessage(np->m_accounts[account], np->adminLegacyName, std::string("Steam Guard code: "));
+		else {
+			LOG4CXX_INFO(logger, "generic input request");
+			np->handleMessage(np->m_accounts[account], np->adminLegacyName, std::string(primary));
+			np->handleMessage(np->m_accounts[account], np->adminLegacyName, std::string(secondary));
```

Comment:
These are only valueable changes or I miss something? :)
What is `np->adminLegacyName` in generic case? It was set up for every listed plugin before. See also my [checklist](https://github.com/SpectrumIM/spectrum2/issues/316#issuecomment-443486106) for 2FA